addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover_tests.js

```diff
@@ -206,6 +209,12 @@ QUnit.test('activity mark done popover mark done with feedback', async function
 QUnit.test('activity mark done popover mark done and schedule next', async function (assert) {
     assert.expect(6);
 
+    const bus = new Bus();
+    bus.on('do-action', null, payload => {
+        if (!payload) {
+            throw new Error("The do-action event should not be triggered when the route doesn't return an action");
```

Comment:
Does that actually crash in qunit without the fix? I'm asking because errors are silent in mocked RPC for example. It's usually preferable to create a dummy assert.step and ensure in the test that the step wasn't executed.

Reply:
Yes, I just tested